src/chapters/prosesses_assets.html.j2

```diff
@@ -3,58 +3,54 @@
   <div class="answer">
 
   {%- set categoriesOfDataPath = [uuids.processes_assetsCUuid, uuids.categoriesofdataQUuid]|reply_path -%}
-  {%- set categoriesOfDataItems = repliesMap[categoriesOfDataPath]|reply_items-%}
-  
-  {%- if categoriesOfDataItems -%}
-
-  <h3> GDPR categorisation of the data </h3>
-  {% for choiceUuid in categoriesOfDataItems %}
-      <ul>
-      <li>
-      {{ km.entities.choices[choiceUuid].label }}
-      </li>
-      </ul>
-      {% endfor %}
-  {%- endif -%}
-
-{%- set listOfProjectPath = [uuids.processes_assetsCUuid, uuids.listOfProjectsQUuid]|reply_path -%}
+  {%- set categoriesOfData = repliesMap[categoriesOfDataPath]|reply_items-%}
+
+{%- set listOfProjectPath = [uuids.Processes_assetsCUuid, uuids.listOfProjectsQUuid]|reply_path -%}
```

Comment:
I don't think this will work well... the variable names should be case-sensitive so this (and other occurences in this file should be `Processes_assetsCUuid`)

Reply:
now: do you want us to fix this right now?

Reply:
I can do that... no problem